preflight_check/config.py

```diff
@@ -0,0 +1,7 @@
+FENCE_TIMEOUT = 60
+FENCE_NODE = "crm_attribute -t status -N '{}' -n terminate -v true"
+BLOCK_IP = '''iptables -{action} INPUT -s {peer_ip} -j DROP;
+              iptables -{action} OUTPUT -d {peer_ip} -j DROP'''
+REMOVE_PORT = "firewall-cmd --zone=public --remove-port={port}/udp"
+ADD_PORT = "firewall-cmd --zone=public --add-port={port}/udp"
+FENCE_HISTORY = "stonith_admin --history={node}"
```

Comment:
Should use `stonith_admin -h {node}` instead, otherwise won't able to detect the fence message.

Error message: (Actually node2 is fenced as expect)
```
[2020/12/21 17:40:53]INFO: Trying to fence node "SLE15-sp2-gm-nwang-crmshreview-node2"
[2020/12/21 17:40:53]INFO: Waiting 60s for node "SLE15-sp2-gm-nwang-crmshreview-node2" reboot...
[2020/12/21 17:40:54]INFO: Node "SLE15-sp2-gm-nwang-crmshreview-node2" will be fenced by "SLE15-sp2-gm-nwang-crmshreview-node1"!
[2020/12/21 17:41:53]ERROR: Target fence node "SLE15-sp2-gm-nwang-crmshreview-node2" still alive
```
Test environment: SLE15SP2
```
# stonith_admin --history=SLE15-sp2-gm-nwang-crmshreview-node2
reboot of SLE15-sp2-gm-nwang-crmshreview-node2 successful: delegate=SLE15-sp2-gm-nwang-crmshreview-node1, client=pacemaker-controld.1187, origin=SLE15-sp2-gm-nwang-crmshreview-node2, last-successful='2020-12-17 16:50:48 +08:00'
1 event found
# stonith_admin -h SLE15-sp2-gm-nwang-crmshreview-node2
Node SLE15-sp2-gm-nwang-crmshreview-node2 last fenced at: Mon Dec 21 18:01:58 2020
```

Reply:
Changed